src/ucp/api/ucp.h

```diff
@@ -4518,17 +4520,29 @@ typedef struct ucp_ep_attr {
      * this name.
      */
     char     name[UCP_ENTITY_NAME_MAX];
+
+    /**
+     * The local sockaddr that this ep is using.
+     */
+    struct sockaddr_storage local_sockaddr;
+
+    /**
+     * The remote sockaddr this ep is connected to.
```

Comment:
Remote socket address this endpoint is connected to. Valid only for endpoints created by connecting to a socket address.

Reply:
@yosefe Done